cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -1,74 +1,65 @@
 :_content-type: ASSEMBLY
 [id='odo-release-notes']
-= `{odo-title}` release notes
+= `{odo-title}` 3.0.0 release notes
 include::_attributes/common-attributes.adoc[]
 :context: odo-release-notes
 
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable changes and improvements in `{odo-title}` version 2.5.0
-
-// #5238
-* Creates unique routes for each component, using `adler32` hashing
-// #5252
-* Supports additional fields in the devfile for assigning resources:
-** cpuRequest
-** cpuLimit
-** memoryRequest
-** memoryLimit
-// #5276
-* Adds the `--deploy` flag to the `odo delete` command, to remove components deployed using the `odo deploy` command:
-+
-[source,terminal]
-----
-$ odo delete --deploy
-----
-// #5237
-* Adds mapping support to the `odo link` command
-// #5279
-* Supports ephemeral volumes using the `ephemeral` field in `volume` components
-// #5270
-* Sets the default answer to `yes` when asking for telemetry opt-in
-// #5260
-* Improves metrics by sending additional telemetry data to the devfile registry
-// #5287
-* Updates the bootstrap image to `registry.access.redhat.com/ocp-tools-4/odo-init-container-rhel8:1.1.11`
-// #5308
-* The upstream repository is available at link:https://github.com/redhat-developer/odo[]
+== Notable changes and improvements in `{odo-title}` version 3.0.0
 
+`odo` 3.0 includes new commands to improve the developer experience, providing the same functionality as before, but in a faster and more efficient manner.
```

Comment:
```suggestion
`odo` 3.0 includes new commands to improve the developer experience. These commands provide the same functionality as before but in a faster and more efficient manner.
```
- Rewrite to clarify the unclear modifier of "providing". As constructed, the sentence isn't clear what is providing the same functionality -- the experience or the commands.
- no comma needed before a conjunction with a dependent clause.